index.rst

```diff
@@ -87,6 +87,57 @@ Catalog Data Access
 We assume that most user analysis cases that involve spatially localized searches of the Object or Source table, and/or the retrieval of ForcedSource data for limited sets of Objects, will go to Qserv via the TAP service.
 
 
+Bulk reprocessing of commissioning data
+---------------------------------------
+
+During Rubin/LSST commissioning it will be necessary to reprocess the data collected to date with evolving versions of the pipelines. 
```

Comment:
I am not sure this is a "user" batch use case, unless you mean individual developers are trying out pipeline modifications outside the normal cycle of Science Pipelines development and test.

Reply:
Apologies, I wasn't clear. We now have a commissioning AO that will bring in a large number of people from the community who will be want to reprocess the commissioning data as part of their contribution towards commissioning. I don't think this topic has been discussed yet but I don't expect it will be covered by the regular reprocessing of commissioning data at the USDF  (which  I agree is not a user batch use case).